selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/wiki/WikiLocators.java

```diff
@@ -0,0 +1,629 @@
+package org.exoplatform.platform.qa.ui.selenium.locator.wiki;
```

Comment:
Please format the code with eXo Formatter Settings http://developer.exoplatform.org/#id-formatter-settings